src/Logger.java

```diff
@@ -0,0 +1,9 @@
+public class Logger {
+
+    public void print(String[] strings) {
+        for (String string : strings) {
```

Comment:
This method does too much. The `logger.print()` call can happen inside a `for` loop, but should not have a `for` loop itself. This method should then only take a `String`, but it's OK to call it multiple times.